apps/events-helsinki/README.md

```diff
@@ -1,5 +1,40 @@
-#Events-Helsinki
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+# Events-Helsinki
+
+Test server: https://tapahtumat.test.hel.ninja
+
+Staging server: https://tapahtumat.stage.hel.ninja
+
+Production server: https://tapahtumat.hel.fi (NOTE: this production server might still be deployed from the old Tapahtumat-repo)
+
+This is a [Next.js](https://nextjs.org/) project originally bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app), but cloned from the Hobbies-Helsinki.
+
+## Architecture
+
+```
+┌─────────────┐               ┌───────────────┐             ┌── Headless CMS         (app specific datasource for the dynamic page and articles content)
+│             │  supergraph > │               │ subgraphs > ├── Unified-Search       (Elasticsearch-service for search results scoring)
+│ Application ├───────────────┤ Apollo Router ├─────────────├── Events GraphQL Proxy (A GraphQL-proxy for the LinkedEvents)
+│             │               │               │             └── Venues GraphQL Proxy (A GraphQL-proxy for the Palvelukartta/Servicemap / "TPREK" Toimipaikkarekisteri)
+└─────────────┘               └───────────────┘
+```
+
+### Headless CMS datasources
+
+- Test / Staging: https://tapahtumat.hkih.stage.geniem.io/graphql ([Wordpress-admin](https://tapahtumat.hkih.stage.geniem.io/wp-admin/))
+- Production: https://tapahtumat.content.api.hel.fi/wp-admin/graphql ([Wordpress-admin](https://tapahtumat.content.api.hel.fi/wp-admin/wp-admin/))
+
+### Route (pages)
+
+The pages are served with some server side rendering (SSR) mechanism to offer better search engine optimization (SEO) and fast user friendly UI.
+The pre-rendering process that we mostly use here is SSG - "automatically generated as static HTML + JSON (uses getStaticProps)".
+More about NextJS's data fetching in https://nextjs.org/docs/basic-features/data-fetching/overview.
+
+There are some rules that a content manager must follow while maintaining the CMS:
+
+1. **/ (The root path / the front page)**: The root path is set as a landing page and it acts as a home page or the front page. The slug for this page must be an empty string, so the uri is `/` in every language.
+2. **/search (The search page)**: The search page gets no content from the CMS, except the SEO information. The uri for this page must be `/search` in every language. It is translated to different languages with the `i18nRoutes.config.json` configuration file.
```

Comment:
There could be a straight link to the `i18nRoutes.config.json` file

Reply:
In same style as `[README](./proxies/events-graphql-federation/README.md)`

Reply:
added